app-shells/mcfly/mcfly-0.9.3.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="Context-aware bash history search replacement (ctrl-r)"
+DESCRIPTION="McFly replaces your default ctrl-r shell history search with an intelligent search \
+engine that takes into account your working directory and the context of recently executed \
+commands. McFly's suggestions are prioritized in real time with a small neural network.
+
+TL;DR: an upgraded ctrl-r where history results make sense for what you're working on right now.
+
+Features
+
+* Rebinds ctrl-r to bring up a full-screen reverse history search prioritized with a small neural \
+network.
+* Augments your shell history to track command exit status, timestamp, and execution directory in \
+a SQLite database.
+* Maintains your normal shell history file as well so that you can stop using McFly whenever you \
+want.
+* Unicode support throughout.
+* Includes a simple action to scrub any history item from the McFly database and your shell \
+history files.
+* Designed to be extensible for other shells in the future.
+* Written in Rust, so it's fast and safe.
+* You can type % to match any number of characters when searching.
+* Supports Zsh, Bash (version 3+), and PowerShell (version 7+)"
```

Comment:
```suggestion
* Supports Zsh, Bash (version 3+), and PowerShell (version 7+)
 
To enable McFly on:

* bash: add `eval "$(mcfly init bash)"` to the end of ~/config/settings/profile.
* zsh: add `eval "$(mcfly init zsh)"` to the end of ~/config/settings/zsh/.zshrc.

And then restart Terminal."
```

Should be enough instructions to get started, I think.
(albeit I'm not sure if it is `[...]/zsh/zshrc` or `[...]/zsh/.zshrc` for Zsh)